### PR TITLE
finished implementation of training on mnist dataset

### PR DESCRIPTION
**Pull request by PhoenixDai**
I finished implementation of training on mnist dataset and it can run now. However, the g_loss is not converging. I need help from other users to improve the program.

### CODE REVIEW

**Comment by carpedm20** [comment]
Cool! How was the `g_loss` and others look like?

**Comment by PhoenixDai** [comment]
I tried this one: https://github.com/Newmu/dcgan_code/tree/master/mnist and it works well. I'm currently trying to compare with that implementation and hope I can got some idea of solving the g_loss issue.

**Comment by PhoenixDai** [comment]
The g_loss issue has been fixed. I think it's ready to merge now. The issue was caused by the images not normalized (by 255 to 0~1 range).

**Comment by tonywu95** [comment]
@PhoenixDai I tried your code for MNIST. Somehow the loss does not go down (stabilize at 1.36). Could you share your hyper-parameter choices?

**Comment by carpedm20** [comment]
@tonywu95 Thanks for the test. I forgot to test this.. but now I can merge this 👍

**Comment by PhoenixDai** [comment]
@carpedm20 Thank you! It's my pleasure. 
@tonywu95 I'm also getting similar results. I also tried the original implementation by the author of DC-GAN and it's also similar. I think this might be what this architecture can learn on this dataset. I'm trying similar tasks with different architecture. If I got better results, I will share it then.
